src/lib/cache/gdfs_cache.hpp

```diff
@@ -132,6 +132,13 @@ class GDFSCache : public AbstractCacheImpl<Key, Value> {
 
   ErasedIterator end() { return ErasedIterator{std::make_unique<Iterator>(_map.end())}; }
 
+  size_t frequency(const Key& key) {
+    const auto it = _map.find(key);
```

Comment:
What if `key` is not part of the `_map`?

Reply:
Actually, returning `0` would be fine in that case as well

Reply:
Returning 0 now, added a test.